src/Carnac.Logic/KeyPlayer.cs

```diff
@@ -3,7 +3,7 @@
 using System.Reactive.Linq;
 using Carnac.Logic.KeyMonitor;
 
-namespace Carnac.Tests
+namespace Carnac.Logic
```

Comment:
The `KeyStreamCapture` project was referencing the `Carnac.Tests` project just for that class, if there is a particular reason for that I'm happy to move it back 😀